requirements.txt

```diff
@@ -3,23 +3,23 @@ decorator==4.0.10
 docutils==0.12
 ebaysdk==2.1.4
 feedparser==5.2.1
-gevent==1.1.2
-greenlet==0.4.10
+gevent>=1.1.2
+greenlet>=0.4.10
 html2text==2016.9.19
 Jinja2==2.8
-lxml==3.5.0
+lxml>=3.5.0
```

Comment:
Constraints such minimal versions supported must be declared in setup.py install_requires.

In a python application, requirements.txt usually expresses "known good / recommended" versions, so it must be `==` everywhere in this file.

Reply:
Ok, I'll revert that but I need it for WinPython, it's easier to use to requirements.txt file than to install missing versions manually. But I think thath it will not be needed in the build process. So will revert that before merging. Thanks for the remark.

Reply:
OTOH, it might be a good time to refresh requirements.txt to the latest version of all the dependencies.